server/app/lib/actions/fusor/deployment/rhev/ose_launch.rb

```diff
@@ -139,6 +153,49 @@ def wait_for_sshd_on_host(host, username, password)
             ::Fusor.log.error "====== waiting for ssh on #{host} TIMED OUT! ======"
             return false
           end
+
+          def add_vda_only_partition_table(ptable_name)
+            default_name = "Kickstart default"
+
+            if Ptable.exists?(:name => ptable_name)
+              ::Fusor.log.debug "====== Partition table '#{ptable_name}' already exists! Nothing to do. ====== "
+              return
+            end
+
+            defaultptable = Ptable.find_by_name(default_name)
+            oseptable = defaultptable.dup
```

Comment:
The default should come with Satellite and should always be available.  However, it makes sense to add an ensure line that will fail the task if it failed to exist for whatever reason.